test/integration.test.ts

```diff
@@ -1,5 +1,36 @@
-describe('placeholder', () => {
-  it('should pass', () => {
-    console.log('yay');
+import {HttpClient} from '../src';
+
+describe('integration test using the Momento HTTP API', () => {
+  it('HttpClient should set, get, and delete cache values', async () => {
+    const cacheName = process.env['MOMENTO_CACHE_NAME'];
+    if (!cacheName) {
+      throw new Error('Missing environment variable MOMENTO_CACHE_NAME');
+    }
+
+    const authToken = process.env['MOMENTO_AUTH_TOKEN'];
+    if (!authToken) {
+      throw new Error('Missing environment variable MOMENTO_AUTH_TOKEN');
+    }
+
+    const endpoint = process.env['MOMENTO_HTTP_ENDPOINT'];
+    if (!endpoint) {
+      throw new Error('Missing environment variable MOMENTO_HTTP_ENDPOINT');
+    }
+
+    const momentoClient = new HttpClient(authToken, endpoint);
+    const key = 'foo';
+    const value = 'bar';
+
+    const setResponse = await momentoClient.set(cacheName, key, value);
+    expect(setResponse).toEqual(204);
```

Comment:
I'm just wondering, why are the responses `204`? I would expect it to be a `200` or `201`

Reply:
204 is usually what is returned from CORs

Reply:
I was following the [docs](https://docs.momentohq.com/develop/api-reference/http-api#set-value) and it said set and delete would return 204 when successful

Reply:
![giphy](https://github.com/momentohq/momento-http-client/assets/11823378/71c4cd36-ec0d-4560-8e23-78b921d28326)